test/user.avro

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
imho, these files can be moved to pkg/codec (codec implementations, and their tests should be there)